packages/firebase_crashlytics/README.md

```diff
@@ -12,43 +12,72 @@ For Flutter plugins for other Firebase products, see [README.md](https://github.
 
 To use the `firebase_crashlytics` plugin, follow the [plugin installation instructions](https://pub.dartlang.org/packages/firebase_crashlytics#pub-pkg-tab-installing).
 
+The following instructions are from [the official installation page](https://firebase.google.com/docs/crashlytics/get-started-new-sdk).
+
 ### Android integration
 
 Enable the Google services by configuring the Gradle scripts as such:
 
-1. Add the Fabric repository to the `[project]/android/build.gradle` file.
-```
-repositories {
-  google()
-  jcenter()
-  // Additional repository for fabric resources
-  maven {
-    url 'https://maven.fabric.io/public'
+1. Check that you have Google's Maven repository in your **project-level** `build.gradle` file (`[project]/android/build.gradle`).
+
+```gradle
+buildscript {
+  repositories {
+    // Add this
+    google()
+
+    // ... you may have other repositories
+  }
+}
+allprojects {
+  repositories {
+    // and this
+    google()
+
+    // ...
   }
 }
 ```
 
-2. Add the following classpaths to the `[project]/android/build.gradle` file.
+2. Add the following classpaths to your **project-level** `build.gradle` file (`[project]/android/build.gradle`).
+
 ```gradle
-dependencies {
-  // Example existing classpath
-  classpath 'com.android.tools.build:gradle:3.2.1'
-  // Add the google services classpath
-  classpath 'com.google.gms:google-services:4.3.0'
-  // Add fabric classpath
-  classpath 'io.fabric.tools:gradle:1.26.1'
+buildscript {
+  dependencies {
+    // Check that you have the Google Services Gradle plugin v4.3.2 or later (if not, add it).
+    classpath 'com.google.gms:google-services:4.3.3'
+    
+    // Add the Crashlytics Gradle plugin.
+    classpath 'com.google.firebase:firebase-crashlytics-gradle:2.0.0'
+
+    // ... you may have other classpaths
+  }
 }
 ```
 
-2. Apply the following plugins in the `[project]/android/app/build.gradle` file.
+3. Apply the following plugins in your **app-level** `build.gradle` file (`[project]/android/app/build.gradle`).
+
 ```gradle
 // ADD THIS AT THE BOTTOM
-apply plugin: 'io.fabric'
 apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.firebase.crashlytics'
 ```
 
-*Note:* If this section is not completed, you will get an error like this:
+4. Add the SDK dependencies in your **app-level** `build.gradle` file (`[project]/android/app/build.gradle`).
+
+```gradle
+dependencies {
+  // Optional but recommended: Add the Firebase SDK for Google Analytics.
+  implementation 'com.google.firebase:firebase-analytics:17.4.0'
+
+  // Add the Firebase SDK for Crashlytics.
+  implementation 'com.google.firebase:firebase-crashlytics:17.0.0'
```

Comment:
This dependency is already in the package, right? We don't need to add it to our Flutter app build.gradle file IMO.

Reply:
Yeah if it's already a dependency used by the plugin then developers shouldn't need to add it to their build.gradle file

cc @axel-op

Reply:
Ditto for the `firebase_analytics` ! I don't like the way the official documentation like this: https://firebase.google.com/docs/flutter/setup?platform=ios

Encourages other packages. It just leads to possible reader confusion.

Reply:
This will be updated when the package will depend on the [new versions of `firebase_core`](https://github.com/FirebaseExtended/flutterfire/issues/2582#issuecomment-653316503).